README.md

```diff
@@ -1,43 +1,48 @@
 # Welcome to TasKing
-## Infastructure
-![Infrastructure overview](https://github.com/Venyla/TasKing/blob/main/doc/TasKing.drawio.png)
```

Comment:
please do not remove this diagram we need to update it

Reply:
Das Bild hani witr une nach em text ihgfüegt.